ballerina/build.gradle

```diff
@@ -895,7 +893,7 @@ task testExamples() {
         def outputFilePath = "${directoryPath}/output.txt"
         def additionalBuildParams = ""
         if (bbe == "secured-service-with-basic-auth" || bbe == "secured-client-with-basic-auth") {
-            additionalBuildParams = "--b7a.config.file=${baseBuildPath}/${bbe}/sample-users.toml"
+            additionalBuildParams = "--b7a.config.file=${distPath}/${bbe}/sample-users.toml"
```

Comment:
This probably need to be updated right? This is a usage of the old config API. @ldclakmal thoughts?

Reply:
Just wondering how these tests were passing, if its an invalid config API shouldn't it fail?

Reply:
Yes. We don't need this anymore. These are old config API usages. With the new `configurable` feature, those test cases were also removed. That's why the tests were passing.